config/database.yml.example

```diff
@@ -1,12 +1,24 @@
+default: &default
+  encoding: utf8
+  adapter: mysql2
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  port: 3306
+  socket: /var/run/mysqld/mysqld.sock
+
 development:
```

Comment:
@alaxalves I think we want to leave this as sqlite3 because mysql2 configurations can be a pain esp for new comers

Reply:
This is the point of this PR, the docker setup I've created doesn't create any pain for newcomers. You can literally setup everything with a simple `docker-compose up` command

Reply:
I think we should give them a choice, some people do not prefer docker and may have no experience prior. Is there a way we can setup so both docker users and manual installers can benefit?

Reply:
I'll create another yaml only for docker and keep the old one for manual installers then

Reply:
But I do not recommend keeping a different database for every environment we have. If we use MariaDB for production we should set it for every environment we have.